src/functions/helper_lookup.ts

```diff
@@ -49,6 +49,11 @@ export function addPivotDependencies(
   const originCellXC = evalContext.__originCellXC?.();
```

Comment:
Not directly related to this PR, but shouldn't we put the `CellPosition`  in the evalcontext rather than the XC ?

Reply:
definitely yes